app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/landing/LandingViewModel.kt

```diff
@@ -198,6 +216,19 @@ class LandingViewModel @Inject constructor(
             )
         }
     }
+
+    /**
+     * If the user state account is changed to an active but not "logged in" account we can
+     * pre-populate the email field with this account.
+     */
+    private fun mapToInternalActionOrNull(
+        activeAccount: UserState.Account,
+    ): LandingAction.Internal.UpdateEmailState? {
+        val activeUserNotLoggedIn = activeAccount.isLoggedIn.not()
+        val noPendingAdditions = authRepository.hasPendingAccountAddition.not()
```

Comment:
For app wide consistency, I believe we have been using `!` over `not`

Reply:
no problem, gotten in the habit of using `.not()` cause I find it stands out more when you are just glancing, but I can adjust!